src/pkg/cli/compose/validation.go

```diff
@@ -413,3 +428,26 @@ func ValidateProjectConfig(ctx context.Context, composeProject *composeTypes.Pro
 
 	return nil
 }
+
+func ValidateManagedStore(managedStore any) error {
+	if managedStore == nil || managedStore == true {
+		return nil
+	}
+
+	if managedStore == false {
+		return ErrManagedStoreParam("to not use managed storage remove the 'x-defang-postgres' or 'x-defang-redis' fields")
```

Comment:
I think it's reasonable to handle `false` as meaning "not managed"; we should support it as an explicit opt-out.

Reply:
I'll check with cd to make sure we ignore when set to false.

Reply:
confirm that false bypasses creating managed resource. Creates a regular container.